package.json

```diff
@@ -1,7 +1,7 @@
 {
-    "name": "theme-gruvbox-dark-medium",
```

Comment:
I'd prefer to keep the namespace since this is a `theme` extension.

Also, for the naming discussion, as in my previous comment, I think we should keep the theme specific, since this theme is only 1 of the 6 variants. Maybe still keep `gruvbox-dark-medium` or `gruvbox-medium-dark`, display name and description should be changed respectively when we come to a decision on this.

Reply:
Ok I understand what you mean. But what if we added the rest of the background colours? We could add way to change it from the settings. What do you think?

Reply:
Yeah, we can have more options for one theme package and if we can add the rest, we can definitely change the name.
However, I don't use those other variants so the best I (and I think you as well) could do is that we create those variants by replacing the colors with the corresponding one from gruvbox's color list.
But that maybe for when we have more time to do. So for this PR, we'll just keep `gruvbox-dark-medium`. We'll change the name later when we add those variants.

Reply:
yeah, i don't use the other ones either. But since we came this far it would be cool to port the full theme. check out the rest of the stuff and give me your opinion